src/tool/mod.rs

```diff
@@ -1,5 +1,25 @@
+use std::{
+    fmt::Display,
+    io::{self, Write},
+};
+
+use anyhow::Context;
+
 pub(crate) mod cli;
 pub(crate) mod config;
 pub(crate) mod config_file;
 pub(crate) mod core;
 pub(crate) mod log;
+
+fn does_user_confirm<S: Display>(prompt: S) -> anyhow::Result<bool> {
```

Comment:
also why not just fail and ask them to use `-f` for forced pull.

Reply:
I'd be ok with having an option to support `-f` but I do also like having the confirmation option because I usually don't know that I already did it and I would prefer press y than rerun the command.